src/axom/quest/interface/internal/QuestHelpers.hpp

```diff
@@ -229,9 +224,44 @@ int read_mesh_shared(const std::string& file,
  * \see STLReader
  * \see PSTLReader
  */
-int read_mesh(const std::string& file,
-              mint::Mesh*& m,
-              MPI_Comm comm = MPI_COMM_SELF);
+int read_stl_mesh(const std::string& file,
+                  mint::Mesh*& m,
+                  MPI_Comm comm = MPI_COMM_SELF);
+
+#ifdef AXOM_USE_C2C
+/*!
+ * \brief Reads in the contour mesh from the specified file
+ *
+ * \param [in] file the file consisting of the surface
+ * \param [in] segmentsPerPiece number of segments to sample per contour Piece
+ * \param [in] vertexWeldThreshold threshold for welding vertices of adjacent curves
+ * \param [out] m user-supplied pointer to point to the mesh object
+ * \param [in] comm the MPI communicator, only applicable when MPI is available
+ *
+ * \note This method currently expects the contours to be given in the C2C format
```

Comment:
Since the function is called `read_c2c_mesh()`, I don't think this will ever change. It can probably be left out. Unless you were trying to specify that it should be a contour file rather than an assembly file.